### PR TITLE
Feature/enchancements

### PR DESCRIPTION
**Pull request by lukachi**

### CODE REVIEW

**Comment by lukachi** [comment]
FIXME:

1. Too much prioritized component styles
2. something wrong with ctc in toasts, e. g. can't access to $t, or $route, so custom components can't be fully implemented in toast popup
3. Modal props warnings
4. Fields emit change & input events twice
5. UICollapse ain't work properly if we call it in fields

File: .env-analyze [hunk 1]
```diff
@@ -0,0 +1,4 @@
+VITE_ENVIRONMENT=analyze
+VITE_PORT=8095
+VITE_API_URL='https://jsonplaceholder.typicode.com'
+VITE_APP_NAME='React Template'
```
Comment (napalmpapalam) [inline .env-analyze]: Vue Template

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "bracketSameLine": false,
+  "arrowParens": "avoid",
+  "endOfLine": "auto",
+  "trailingComma": "all",
+  "printWidth": 100,
+  "quoteProps": "as-needed",
+  "embeddedLanguageFormatting": "auto",
+  "htmlWhitespaceSensitivity": "css",
+  "jsxSingleQuote": true,
+}
```
Comment (napalmpapalam) [inline .prettierrc]: Half of the options have default values

```suggestion
{
  "semi": false,
  "singleQuote": true,
  "arrowParens": "avoid",
  "endOfLine": "auto",
  "printWidth": 100,
  "jsxSingleQuote": true,
}
```

File: src/ui/UiCheckbox.vue [hunk 3]
```diff
@@ -1,72 +1,75 @@
...
-}>()
+const attrs = useAttrs()
+
+const isDisabled = computed(() =>
+  ['', 'disabled', true].includes(attrs.disabled as string | boolean),
+)
+
+const isReadonly = computed(() =>
+  ['', 'readonly', true].includes(attrs.readonly as string | boolean),
+)
```
Comment (napalmpapalam) [inline src/ui/UiCheckbox.vue]: extract tuples into the constants

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

**Pull request merged**